src/components/Cards/components/Card/styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+export const Card = styled.div`
+  border: 2px solid #aaa;
+  flex-direction: column;
+  border-radius: 1rem;
+  font-size: 1.3rem;
+  padding: 0.5rem;
+  display: flex;
+`;
+
+export const ImageWrapper = styled.div`
+  display: flex;
+`;
+
+export const Paragraph = styled.p``;
```

Comment:
But is it ok to leave it like that? For future styling for example. As i remember there was empty tags in the previos project

Reply:
that project were done not in a good or proper way. It were done for 4 hours, and code style there should be enhanced. 
Future styling can never happen, so remove and add when it will be necessary.